drivers/base/bootvid/precomp.h

```diff
@@ -41,17 +49,20 @@ typedef struct tagBITMAPINFOHEADER
 #define BI_RGB  0
 #define BI_RLE4 2
 
-typedef struct _PALETTE_ENTRY
-{
-    UCHAR Red;
-    UCHAR Green;
-    UCHAR Blue;
-} PALETTE_ENTRY, *PPALETTE_ENTRY;
+typedef ULONG RGBQUAD;
+
+#define RGB(r,g,b)          ((RGBQUAD)(((UCHAR)(b)|((USHORT)((UCHAR)(g))<<8))|(((ULONG)(UCHAR)(r))<<16)))
```

Comment:
```suggestion
#define RGB(r, g, b)    ((RGBQUAD)(((UCHAR)(b) | ((USHORT)((UCHAR)(g))<<8)) | (((ULONG)(UCHAR)(r))<<16)))
```
(to improve the readability)